cmd/kube-proxy/app/server_others.go

```diff
@@ -110,6 +114,26 @@ func newProxyServer(
 		return nil, err
 	}
 
+	// create client for each tenant partition, if applicable
+	var clients []clientset.Interface
+	if len(config.TenantPartitionKubeConfig) == 0 {
```

Comment:
for your consideration: it probably will be clearer to move this loop to the caller, so to keep the proxyServerer type unchanged.